packages/compartment-mapper/test/scaffold.js

```diff
@@ -560,22 +586,25 @@ export function scaffold(
       t.plan(1);
       await setup();
 
-      const expectedSha512 = await hashLocation(readPowers, fixture, {
-        modules,
-        Compartment,
-        conditions: new Set(['development', ...(conditions || [])]),
-        strict,
-        searchSuffixes,
-        commonDependencies,
-        parserForLanguage,
-        languageForExtension,
-        commonjsLanguageForExtension,
-        moduleLanguageForExtension,
-        workspaceLanguageForExtension,
-        workspaceCommonjsLanguageForExtension,
-        workspaceModuleLanguageForExtension,
-        ...additionalOptions,
-      });
+      const expectedSha512 = await hashLocation(
+        /** @type {HashPowers} */ (readPowers),
```

Comment:
This is to note that the type signature for `makeReadPowers` is insufficient; the return type depends on the parameters.  If, for example, a `crypto: CryptoInterface` object is provided, then the return type should extend `HashPowers`.  If it contains both `fs: FsInterface` and `url: UrlInterface`, then the return type should extend `ReadNowPowers`.  The return type always extends `MaybeReadPowers`.  And combinations thereof